mods/persistence/modules/mob/living/simple_animal/friendly/mushroom.dm

```diff
@@ -0,0 +1,20 @@
+/mob/living/simple_animal/mushroom/before_save()
```

Comment:
Aside note: We really need to eliminate any of the 'mob' hydroponics plants. Clogging up the mobs list with useless dead mobs is a pain and they're pretty immersion breaking too.

Reply:
I tend to disagree with that. I'm not really sure where you're even coming from with "immersion breaking", since we'd need to have an immersion to break in the first place. Plus if we keep removing things for any odd reasons we're gonna have even less things to do than we have now, which is really not much at all.

Also, clogging the dead mob list is gonna be an issue regardless, we'll have to work some sort of pruning at one point. Probably just turning mobs dead for a certain time into husk objects/effects or something..

Reply:
Killer tomatoes with evil faces don't really belong in a serious setting in my opinion. And yes, we do need to work on a more general pruning method even for human mobs, so best let it be for the moment. In previous testing period thought players did spam grow killer tomatoes, so it needs to be resolved somehow.

Reply:
Just make them live for a very short time. The thing is they got their uses in cooking among a few other things. I honestly don't think its much of a problem to have goofy things like that. Especially considering how everything else is already very eyebrow raising.  Its just meant to create conflicts and shennanigans, where there would otherwise not be any, botany being one of the less eventful part of the game.